omero/sysadmins/windows/install-web.txt

```diff
@@ -4,6 +4,12 @@ OMERO.web deployment
 .. Warning:: The guidance below applies to OMERO **5.2**. Code fixes for
     OMERO.web deployment on Windows have not been back-ported to the 5.1 line
     and this guide cannot be used to deploy OMERO.web 5.1.x on Windows.
+    
```

Comment:
does it make sense to split http://www.openmicroscopy.org/site/support/omero5.2-staging/sysadmins/windows/install-web.html and make windows removal first ?